pkg/volume/util/metrics.go

```diff
@@ -80,6 +82,17 @@ var storageOperationEndToEndLatencyMetric = metrics.NewHistogramVec(
 	[]string{"plugin_name", "operation_name"},
 )
 
+var csiOperationsLatencyMetric = metrics.NewHistogramVec(
+	&metrics.HistogramOpts{
+		Subsystem:      "csi_kubelet",
```

Comment:
add this as a const instead so it's clear what to use if more csi metrics are added.

And it doesn't look like we have any subsystems defined. I think it might be simpler to just have a "csi" subsystem? eg, the existing storage_operation_duration_seconds metric doesn't have one version for the master and one for the kubelet, the same metric is used both places. I think it would be a good idea to be consistent with that for CSI metrics too.

Unless there's some other context here for a csi_kubelet subsystem?

Reply:
I am doing the sub_system thing here because the current sidecar CSI metric name is: `csi_sidecar_operations_seconds`, I dont think the sidecar part fits in here. So I am coming up with this name. I am fine with `csi_operations_seconds` here as well. But note that this metrics only exists for the kubelet part and kube-controller-manager is not exposing this metric at all.


https://github.com/kubernetes-csi/csi-lib-utils/blob/44b91e2a7b0af8199fc65c1142f664fd5f7abb96/metrics/metrics.go#L39